docs/content/examples/tutorials/mailserver-behind-proxy.md

```diff
@@ -105,7 +105,7 @@ and to `docker-data/dms/config/postfix-master.cf`:
 
 ```cf
 submission/inet/smtpd_upstream_proxy_protocol=haproxy
-submissions/inet/smtpd_upstream_proxy_protocol=haproxy
+smtps/inet/smtpd_upstream_proxy_protocol=haproxy
```

Comment:
You are looking at the `:edge` docs, `v12.1` will show `smtps`.

`submissions` is correct within the container. It can be found in `/etc/services` for port 465 (_submission(s), the last `s` is akin to http(s), using TLS_).

`smtps` is obsolete name for when it was briefly assigned for implicit TLS over port 465 as an alternative to port 25 SMTP, but since 2018 is known as Submissions port akin to the Submission port 587. That is, this port is only for authenticated mail submission.

Any information that you come across that says otherwise is misinformed.

Reply:
To be honest, I'm not that deep in this topic. I was just setting up the mailserver and I wasn't able to connect to the smtp server via port 465 with this lines (SSL routines:ssl3_get_record:wrong version number and the IP was showing the proxy address). After changing the line as proposed it worked.

Reply:
That's ok, this is due to an unreleased change. Your docs version just wasn't matching your image version, hence the mismatch for you causing that problem 😅